<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="vinz" failures="1" tests="5" name="LearningSelenium" time="0.344" errors="0" timestamp="2021-11-15T12:38:23 MST">
  <testcase classname="org.learning.com.MyTest1" name="test1" time="0.0"/>
  <testcase classname="org.learning.com.MyTest1" name="test1" time="0.0"/>
  <testcase classname="org.learning.com.MyTest1" name="test1" time="0.0"/>
  <testcase classname="org.learning.com.MyTest1" name="test2" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.learning.com.MyTest1.test2(MyTest1.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="org.learning.com.MyTest1" name="test3" time="0.0">
    <skipped/>
  </testcase> <!-- test3 -->
</testsuite> <!-- LearningSelenium -->
